content/guides/box-ai/ask-questions.md

```diff
@@ -0,0 +1,54 @@
+---
+rank: 3
+related_endpoints:
+  - post_ai_ask
+---
+
+# Ask questions to Box AI
+
+<Message type="notice">
+Box AI API is a beta feature, which means the
+available capabilities may change.
+Box AI API is available to all **Enterprise Plus** customers.
+
+</Message>
+
+Box AI API allows you to
+ask a question about a supplied file or
+a set of files, and get a response based on
+the content.
+For example, while viewing a document in Box,
```

Comment:
Actually the API doesn't require viewing the content. We can improve that lates

Reply:
I reworked it after legal review.